unittests/CppInterOp/InterpreterTest.cpp

```diff
@@ -58,6 +58,9 @@ TEST(InterpreterTest, DebugFlag) {
 }
 
 TEST(InterpreterTest, Evaluate) {
+#ifdef EMSCRIPTEN
+  GTEST_SKIP() << "Test fails for Emscipten builds";
+#endif
```

Comment:
Hmmm technically we should be able to test this in the browser. 

What error are you encountering when you do this.

I see this through xeus-cpp-lite. We use c++20 hence, we have `202002` and I tried the other Evaluate thingy too

![image](https://github.com/user-attachments/assets/a6fff114-3340-4a5d-a46f-c999eb45a8ee)